randovania/games/prime1/json_data/Magmoor Caverns.txt

```diff
@@ -530,10 +566,12 @@ Extra - asset_id: 207070785
               Heat Runs (Expert) or Heat-Resisting Suit
               Any of the following:
                   All of the following:
-                      Morph Ball Bomb and Morph Ball and Bomb Jump (Advanced)
+                      Morph Ball Bomb and Morph Ball and Bomb Jump (Intermediate)
```

Comment:
Any time you actually change logic, please update `CHANGELOG.MD` to reflect the change.